packages/gatsby-theme-newrelic/src/utils/track.js

```diff
@@ -0,0 +1,13 @@
+function track(fn, handler, before) {
```

Comment:
Can we update this to use the ES6 arrow function format?

Also, we may want to include some comments documenting how this function works and what each parameter does. If you use the [JSDoc](https://jsdoc.app/tags-param.html) format, then VSCode will provide some really nice hints when utilizing the function.